definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -6333,7 +6291,7 @@
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "boRequestTranslationFromWLU",
     "UserRole": "caseworker-divorce-solicitor",
-    "CRUD": "CRU"
+    "CRUD": "R"
```

Comment:
this should also go to feature toggled file?

Reply:
Update: 7e02721a3a481b6fe20c0e00ed706a9b58d1f0fe